pbrd/pbr_map.c

```diff
@@ -357,6 +357,9 @@ extern void pbr_map_delete(struct pbr_map_sequence *pbrms)
 	if (pbrms->nhg)
 		pbr_nht_delete_individual_nexthop(pbrms);
 
+	XFREE(MTYPE_TMP, pbrms->nhgrp_name);
```

Comment:
Isn't this already freed by `pbr_map_delete_nexthops()`?

Reply:
During `pbr_map_delete()`, `pbr_map_delete_nexthops()` only gets called `if (pbrms->nhg)`